CHANGELOG.md

```diff
@@ -10,7 +10,9 @@ and this project adheres to
 
 ## Added
 
+- Add route detailing current user
 - Add related certificate products to enrollment API endpoint
+- Remove djmoney and Moneyfields
```

Comment:
actually I moved this item to the `Removed` section in my previous merged PR. You can find it below so this is a duplicate.